src/screens/tabs/Profile/modals/ImportKey/index.tsx

```diff
@@ -75,9 +82,9 @@ function ImportKey({ createImportRef, modalRef, accountsModalRef }: Props) {
           style={styles.inputStyle}
           onChangeText={handleOnChangeKey}
         />
-        {error && (
+        {errorType && (
           <Text type="caption" style={styles.error}>
-            {t('createImportAccount.invalidKey')}
+            {t(`createImportAccount.${errorType}`)}
```

Comment:
can you explain me the logic behind this? and the `eType`

Reply:
basically now the validatePem function returns errors, we can have 2 errors:
1) invalid-key
2) account-added

So when I get the error from the action, I pass it to camelCase so i can use it to get the proper text to show as error